src/os/bluestore/HybridAllocator.cc

```diff
@@ -172,7 +172,7 @@ void HybridAllocator::init_rm_free(uint64_t offset, uint64_t length)
           bmap_alloc->init_rm_free(o, l);
         } else {
           lderr(cct) << "init_rm_free lambda " << std::hex
-            << "Uexpected extent: "
+            << "Unexpected extent: "
```

Comment:
Same as above

Reply:
I'm not sure what _same as above_ means here, sadly GitHub has a number of different views and it's hard to keep track. I _think_ this is "it's debug, it's ok to change w/o a note".

Reply:
correct.